docs/docs/data/create.tsx

```diff
@@ -534,6 +534,12 @@ const createEndpointTableData = [
         </code>
       </>
     )
+  },
+  {
+    "name": "maxNumPages",
+    "required": false,
+    "type": "Number",
+    "description": (<>Overrides the default maximum number of pages that an uploaded presentation can contain defined in <code>bigbluebutton.properties</code>.</>)
   }
```

Comment:
```suggestion
  {
    "name": "maxNumPages",
    "required": false,
    "type": "Number",
    "default": 200,
    "description": (<>Maximum number of pages allowed for an uploaded presentation.</>)
  }
```

I don't believe it's necessary to mention that it overrides `bigbluebutton.properties`, as similar `/create` parameters like `maxParticipants`, `logoutURL`, `duration`, and `welcome` don't include this detail either.  

I also added the default value.